events/README.md

```diff
@@ -10,7 +10,7 @@ What meetings, and regular events do we have.
 | [Artsy Learns to Code](/events/artsy-learns-to-code.md#readme) | Experimental structure where we teach non-engineering colleagues how to code. |
 | [Demo Days](/events/demo-days.md#readme) | How we showcase shipped projects |
 | [Happy Hours](/events/happy-hour.md#readme) | When all of Artsy comes together with partners, friends, artists and engineers. |
-| [Knowlege Share](/events/knowledge-share.md) | A meetup where we share knowledge formally and informally across the org. |
+| [Knowledge Share](/events/knowledge-share.md#readme) | A meetup where we share knowledge formally and informally across the org |
```

Comment:
This is coming from https://github.com/artsy/README/blob/da1046f43c31b855b69dcc0e58f83ce84785dd78/scripts/create-readmes.ts#L17 which we run as a pre-commit hook to generate the table above